java/EpochWriter.java

```diff
@@ -124,9 +134,9 @@ public EpochWriter(
 	// Returns true to continue processing, or false if endTime has been reached
 	public boolean newValues(
```

Comment:
@chanshing - can you add in a comment here to very explicitly state what the variable "long time" expects?